tests/test_resources/test_clusters/test_multinode_cluster.py

```diff
@@ -27,3 +27,12 @@ def test_rsync_and_ssh_onto_worker_node(self, multinode_cpu_cluster):
         assert status_codes[0][0] == 0
 
         assert "runhouse" in status_codes[0][1]
+
+    @pytest.mark.level("thorough")
+    def test_ray_started_on_worker_node_after_cluster_restart(
+        self, multinode_cpu_cluster
+    ):
+        head_node = multinode_cpu_cluster.ips[0]
+
+        status_codes = multinode_cpu_cluster.run(["ray status"], node=head_node)
+        assert status_codes[0][0] == 0
```

Comment:
We should check that two nodes show up in the output, right?

Reply:
Does .run() return the console output?